.github/workflows/cron.yml

```diff
@@ -0,0 +1,186 @@
+on:
+  push:
+  # schedule:
+  #   - cron: 0 0 * * * # at the end of each day
+
+jobs:
+  check_for_public_api_diffs:
+    runs-on: ubuntu-latest
+    permissions: write-all
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: zcash/librustzcash
+          path: librustzcash
+          fetch-depth: 0
+          fetch-tags: true
+
+      - uses: actions/checkout@v4
+        with:
+          repository: eigerco/uniffi-zcash-lib
+          path: uniffi-zcash-lib
```

Comment:
If I don't add `ref: feature/cli-diff-tool` and `ctoyan`, instead of `eigerco`, this PR won't pass.
The other option is to extract the scripts folder and merge it first and them merge this workflow.

[This is the error](https://github.com/ctoyan/uniffi-zcash-lib/actions/runs/6640257614/job/18040309664#step:6:11) that's happening and it's because there is no scripts folder in the main branch of the repo.

Reply:
@ctoyan is this solved?